### PR TITLE
WinUI-3213 : Getting started for SfMaskedTextBox control.

### PR DESCRIPTION
**Pull request by Vignesh-SF3580**
### Description
UG document for SfMaskedTextBox control.

### Solution Description
Updated the getting started file for the masked text box control.
Added the codes and images for simple and regex masks.

### CODE REVIEW

File: winui/Masked-TextBox/Getting-Started.md [hunk 1]
```diff
@@ -7,4 +7,85 @@ control: SfMaskedTextBox
...
+
+## Creating an application with WinUI MaskedTextBox control
+
+1. Create a [WinUI 3 desktop app for C# and .NET 5](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/get-started-winui3-for-desktop).
+2. Add reference to [Syncfusion.Editors.WinUI](https://www.nuget.org/packages/Syncfusion.Editors.WinUI) NuGet. 
+3. Import the control namespace `Syncfusion.UI.Xaml.Editors` in XAML or C# code.
+4. Initialize the `MaskedTextBox` control.
+
+## Initialize MaskedTextBox control using Simple mask
+
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Getting-Started.md:22]: @Vignesh-SF3580 Initialize the MaskedTextBox control using simple mask

File: winui/Masked-TextBox/Getting-Started.md [hunk 2]
```diff
@@ -7,4 +7,85 @@ control: SfMaskedTextBox
...
+// Creating an instance of the MaskedTextBox control.
+SfMaskedTextBox maskedTextBox = new SfMaskedTextBox();
+maskedTextBox.MaskType = MaskedTextBoxMaskType.Simple;
+maskedTextBox.Mask = "00/00/0000";
+
+{% endhighlight %}
+{% endtabs %}
+
+![MaskedTextBox control overview in WinUI](MaskedTextBox_Images/winui_getting_started_simple_mask.png)
+
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Getting-Started.md:56]: @Vignesh-SF3580 Simple mask in WinUI MaskedTextBox
also change winui_simple_mask.png

File: winui/Masked-TextBox/Getting-Started.md [hunk 3]
```diff
@@ -7,4 +7,85 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Getting Started with WinUI SfMaskedTextBox
\ No newline at end of file
+# Getting Started with WinUI SfMaskedTextBox
+
+This section explains the steps required to add the WinUI MaskedTextBox control and covers the basic features needed to get started with Syncfusion `MaskedTextBox` control.
+
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Getting-Started.md:13]: @Vignesh-SF3580 This section explains the steps required to add the WinUI MaskedTextBox control to get started with Syncfusion `MaskedTextBox` control.

File: winui/Masked-TextBox/Getting-Started.md (hunk 1 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Getting-Started.md:22]: Changed the sentence like above.

File: winui/Masked-TextBox/Getting-Started.md (hunk 2 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Getting-Started.md:58]: Changed the image content and image name.

File: winui/Masked-TextBox/Getting-Started.md (hunk 3 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Getting-Started.md:13]: Changed the content like above.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**